minard/detector_state.py

```diff
@@ -121,6 +121,9 @@ def get_detector_state_check(run=0):
         hv_relay_mask1 = detector_state[crate]['hv_relay_mask1']
         hv_relay_mask2 = detector_state[crate]['hv_relay_mask2']
 
+        readout_mask = detector_state[crate]['xl3_readout_mask']
+        readout_mask = readout_mask ^ 0xffff
```

Comment:
I would just add a check before here to see if the readout mask is None and if so to warn that the readout mask is unknown.